Assets/00_Content/Scripts/Rounds/RoundController.cs

```diff
@@ -71,33 +79,36 @@ public class RoundController : SingletonBehaviour<RoundController> {
 			VikingController.Instance.StatScaling = new VikingScaling(difficulty, currentRound);
 		}
 
+		private void DisableGamePlay() {
+			VikingController.Instance.CanSpawn = false;
```

Comment:
Your not checking for the instance here, might want to?

Reply:
Yeah, done!